spec/acceptance/01_agent_spec.rb

```diff
@@ -92,6 +92,56 @@ class { 'sensu::agent':
     end
   end
 
+  context 'etc_dir changed' do
+    it 'should work without errors' do
+      pp = <<-EOS
+      class { '::sensu':
```

Comment:
same here with ensuring that `/etc/sensu` no longer exists

Reply:
per other comment we are already testing that.

```
'api-port' => 4041,
```

We test that the port we define in configs is actually being used and listening, if the config wasn't being used the default port would get used which is different port number.